CONTRIBUTING.rst

```diff
@@ -1,24 +1,21 @@
-# Contributor Guide
+Contributor Guide
+=================
 
 Thank you for your interest in improving this project.
-This project is open-source under the [MIT license] and
+This project is open-source under the `MIT license <https://opensource.org/licenses/MIT>`_  and
 welcomes contributions in the form of bug reports, feature requests, and pull requests.
 
 Here is a list of important resources for contributors:
 
-- [Source Code]
-- [Documentation]
-- [Issue Tracker]
-- [Code of Conduct]
+- `Source Code <https://github.com/21cmfast/tuesday>`_
+- `Documentation <https://tuesday.readthedocs.io/>`_
+- `Issue Tracker <https://github.com/21cmfast/tuesday/issues>`_
+- :ref:`CODE OF CONDUCT`
 
-[mit license]: https://opensource.org/licenses/MIT
-[source code]: https://github.com/21cmfast/py21cmFAST-tools
-[documentation]: https://py21cmFAST-tools.readthedocs.io/
-[issue tracker]: https://github.com/21cmfast/py21cmFAST-tools/issues
+How to report a bug
```

Comment:
I think the rest of the sections should be marked by dashes `-` not equals `=` so that they are sub-sections.